From 56c2b2ee38efc4dae4f7b8c39db0471786e53b24 Mon Sep 17 00:00:00 2001
From: Niko Tyni <ntyni@debian.org>
Date: Wed, 1 Jul 2015 22:23:53 +0300
Subject: Adjust manual page sections in ExtUtils::MakeMaker for the Debian
 Perl policy.

This was inherited from early changes by Brendan O'Dea, previously
in debian/prefix_changes.diff and before that in debian/extutils_hacks.diff

Patch-Name: debian/man-sections.diff
---
 cpan/ExtUtils-MakeMaker/lib/ExtUtils/MM_Any.pm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cpan/ExtUtils-MakeMaker/lib/ExtUtils/MM_Any.pm b/cpan/ExtUtils-MakeMaker/lib/ExtUtils/MM_Any.pm
index 4c00129..88d0896 100644
--- a/cpan/ExtUtils-MakeMaker/lib/ExtUtils/MM_Any.pm
+++ b/cpan/ExtUtils-MakeMaker/lib/ExtUtils/MM_Any.pm
@@ -919,7 +919,7 @@ END
     foreach my $section (qw(1 3)) {
         my $pods = $self->{"MAN${section}PODS"};
         my $p2m = sprintf <<CMD, $] > 5.008 ? " -u" : "";
-	\$(NOECHO) \$(POD2MAN) --section=$section --perm_rw=\$(PERM_RW)%s
+	\$(NOECHO) \$(POD2MAN) --section=\$(MAN${section}EXT) --perm_rw=\$(PERM_RW)%s
 CMD
         push @man_cmds, $self->split_command($p2m, map {($_,$pods->{$_})} sort keys %$pods);
     }
